lib/policyEvaluator/RequestContext.js

```diff
@@ -579,6 +582,27 @@ class RequestContext {
         this._securityToken = token;
         return this;
     }
+
+    /**
+     * Get the policy arn
+     *
+     * @param {string} policyArn - Policy arn
+     * @return {RequestContext} itself
```

Comment:
return the arn :)

Reply:
- [x] update doc with good return type and name